### PR TITLE
Fill color for data explorer box plot

### PR DESCRIPTION
**Pull request by tongyu-microsoft**
This PR fills the color for data explorer box plot, so it can be consistent with new model overview box plots.

## Description
Before:
image

After:
image

For dark mode:
image

## Checklist

- [x] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: libs/dataset-explorer/src/lib/getDatasetBoxOption.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export function getDatasetBoxOption(plotlyProps: IPlotlyProperty): any {
     boxGroupData.push({
       color: data.color,
       data,
+      fillColor: "#b2d6f2",
```
Comment (xuke444) [inline libs/dataset-explorer/src/lib/getDatasetBoxOption.ts:19]: are we able to pick sematic color?

**Comment by romanlutz** [comment]
Do you have a screenshot for dark mode?

**Comment by tongyu-microsoft** [comment]
> Do you have a screenshot for dark mode?

@romanlutz Added screenshot for dark mode in the pr description :D

**Review by imatiach-msft — APPROVED** [review]

**Comment by romanlutz** [comment]
> > Do you have a screenshot for dark mode?
> 
> @romanlutz Added screenshot for dark mode in the pr description :D

Thanks @tongyu-microsoft ! Somehow I had missed this in the model overview as well. This doesn't look great in dark mode. Obviously, I should have caught that earlier 🙁 I took a look at the Fluent UI page but it's not obvious to me which colors change in dark mode to be darker. Any idea?

**Comment by tongyu-microsoft** [comment]
> > > Do you have a screenshot for dark mode?
> > 
> > 
> > @romanlutz Added screenshot for dark mode in the pr description :D
> 
> Thanks @tongyu-microsoft ! Somehow I had missed this in the model overview as well. This doesn't look great in dark mode. Obviously, I should have caught that earlier 🙁 I took a look at the Fluent UI page but it's not obvious to me which colors change in dark mode to be darker. Any idea?

@Romanlutz Updated to a deeper color. How do you think of the new screenshots in the pr description?

**Pull request merged**